src/main/java/org/datavyu/views/VideoController.java

```diff
@@ -1522,9 +1522,10 @@ public void setRegionOfInterestAction() {
     @SuppressWarnings("unused")  // Called through actionMap
     public void goBackAction() {
         try {
-            long oldTime = -CLOCK_FORMAT.parse(goBackTextField.getText()).getTime();
-            logger.info("Jump back by " + oldTime);
-            clockTimer.setForceTime(oldTime);
+            long backTime = -CLOCK_FORMAT.parse(goBackTextField.getText()).getTime();
+            long newTime = (long) clockTimer.getClockTime() + backTime;
```

Comment:
Please check if we need to guard against jumping beyond the min/max here?

Reply:
I did check, setForceTime check if the new time is in between the min and max before seeking.